aws_signing_helper/pkcs11_signer.go

```diff
@@ -286,13 +286,17 @@ func getCertsInSession(module *pkcs11.Ctx, slotId uint, session pkcs11.SessionHa
 		// Fetch the CKA_ID and CKA_LABEL of the matching cert(s), so
 		// that they can be used later when hunting for the matching
 		// key.
-		crtAttributes[0] = pkcs11.NewAttribute(pkcs11.CKA_ID, 0)
+		crtAttributes = []*pkcs11.Attribute{
```

Comment:
What's the implication for memory? I suspect that it's really minor, but I don't know the Go memory model well enough.

Reply:
Yeah, I think it's minor. The object that gets lost through the assignment here (since it can no longer be referenced later in the code) should be garbage collected later on.